src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,25 @@
+package lotto;
+
+public class Lotto {
+	private final LottoNumbers lottoNumbers;
+
+	public Lotto() {
+		this.lottoNumbers = new LottoNumbers(LottoNumberFactory.create());
+	}
```

Comment:
지금 상황에서는 Lotto() 생성자를 호출하는 로직은 테스트가 어렵다는 단점이 있는데요.
LottoNumberFactory로 로또의 생성의 책임을 분리하였는데 내부에서 다시 합쳐진 것 같아요 
Lotto 내부에서 LottoNumberFactory를 호출하는 것보다 외부에서 호출하도록 분리해보면 어떨까요? :)

```suggestion
	public Lotto(LottoNumbers lottoNumbers) {
		this.lottoNumbers = new LottoNumbers(lottoNumbers);
	}
```